pkg/security/secl/model/model_unix.go

```diff
@@ -754,3 +755,82 @@ type RawPacketEvent struct {
 	CaptureInfo gopacket.CaptureInfo `field:"-"`
 	Data        []byte               `field:"-"`
 }
+
+// NetworkStats is used to record network statistics
+type NetworkStats struct {
+	DataSize    uint64 `field:"data_size"`    // SECLDoc[data_size] Definition:`Amount of data transmitted or received`
+	PacketCount uint64 `field:"packet_count"` // SECLDoc[packet_count] Definition:`Count of network packets transmitted or received`
+}
+
+// Add the input stats to the current stats
+func (ns *NetworkStats) Add(input NetworkStats) {
+	ns.DataSize += input.DataSize
+	ns.PacketCount += input.PacketCount
+}
+
+// Flow is used to represent a network 5-tuple with statistics
+type Flow struct {
+	Source      IPPortContext `field:"source"`      // source of the network packet
+	Destination IPPortContext `field:"destination"` // destination of the network packet
+	L3Protocol  uint16        `field:"l3_protocol"` // SECLDoc[l3_protocol] Definition:`L3 protocol of the network packet` Constants:`L3 protocols`
+	L4Protocol  uint16        `field:"l4_protocol"` // SECLDoc[l4_protocol] Definition:`L4 protocol of the network packet` Constants:`L4 protocols`
+
+	Ingress NetworkStats `field:"ingress"` // SECLDoc[ingress] Definition:`Network statistics about ingress traffic`
+	Egress  NetworkStats `field:"egress"`  // SECLDoc[egress] Definition:`Network statistics about egress traffic`
+}
+
+// NetworkFlowMonitorEvent represents a network flow monitor event
+type NetworkFlowMonitorEvent struct {
+	Device                NetworkDeviceContext `field:"device"`      // network device on which the network flows were captured
+	FlowsCount            uint64               `field:"flows_count"` // SECLDoc[flows_count] Definition:`Number of captured network flows`
```

Comment:
Is this field really needed? A `network_flow_monitor.flows.length` is already present